packages/formstr-app/src/containers/CreateForm/components/Sidebar/index.js

```diff
@@ -0,0 +1,26 @@
+import { Layout, Divider } from 'antd';
+import BasicMenu from "../BasicMenu";
+import InputsMenu from "../InputsMenu";
+import PreBuiltMenu from "../PreBuiltMenu";
+import "./index.css";
+
+function Sidebar() {
+  return (
+    <Layout.Sider
+      className="create-sidebar"
+      width={252} 
+      style={{ 
+        height: "calc(100vh - 112px)",
```

Comment:
any reason we are calculating height here?

Reply:
yes, we don't want to show scroll on a page, so our main containers height would be 100vh - 64(header) - 48(footer).